products/rancher.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Rancher
+category: server-app
+tags: suse
+iconSlug: rancher
+permalink: /rancher
+alternate_urls:
+-   /suse-rancher
+releasePolicyLink: https://www.suse.com/lifecycle#rancher
+releaseDateColumn: General Availability
+activeSupportColumn: End of Maintenance
+eolColumn: End of Life
+eolWarnThreshold: 121
+
+identifiers:
+-   purl: docker:rancher/rancher
+
+changelogTemplate: https://github.com/rancher/rancher/releases/tag/v__LATEST__
```

Comment:
This line should be moved after `releasePolicyLink` as per [our style guide](https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide).